src/fuse.js

```diff
@@ -84,6 +84,18 @@
     var key
     var keys
 
+    if (list.constructor === Array) this.typeof = 'array'
+    else this.typeof = typeof list;
+
+    if(this.typeof === 'object') {
+      list = Object.keys(list).map(function (i) {
```

Comment:
Hey, I currently use this library as part of a cross platform mobile app, JavaScriptCore on iOS and v8 on Android, not entirely sure if polyfill would break support for it. 

Not too critical just my two cents in  terms of browser support.